src/features/student/locales/en.json

```
@@ -275,7 +275,13 @@
         }
       },
       "studentSkillView": {
-        "title": "Skill {skill}"
+        "title": "Skill {skill}",
+        "skillEvaluate": {
+          "title": "MY EVALUATION"
+        },
+        "skillProgress": {
+          "title": "MY PROGRESS"
+        }
```

```suggestion
        "evaluation": {
          "title": "My evaluation"
        },
        "progress": {
          "title": "My progress"
        },
        "title": "Skill {skill}"
```

Ordre alphabétique + tu pourras mettre un .toUppercase() plutôt que de mettre l'uppercase ici + t'es pas obligé de mettre skill en préfix de evaluation et progress